src/rvary_fmt_plug.c

```diff
@@ -53,80 +53,80 @@ john_register_one(&fmt_rvary);
 #endif
 
 static const unsigned char a2e[256] = {
-	0,  1,  2,  3, 55, 45, 46, 47, 22,  5, 37, 11, 12, 13, 14, 15,
```

Comment:
Ouch. I never suggested reindenting the whole file, let alone to conventions inconsistent with what we use in the rest of the tree.

I only asked you to keep indentation of the edited blocks consistent, so that it reflects their logical nesting level.